common/modal/modal-component.js

```diff
@@ -36,7 +36,18 @@ const createModal = () => {
 
   // adding close modal button
   const closeButton = createElement('button', ['modal-close-button']);
-  const closeIcon = createElement('i', ['fa', 'fa-close']);
+  const closeIcon = createElement('span', ['icon', 'icon-close']);
+  const svgCloseIcon = document.createRange().createContextualFragment(`
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <g id="Icons/16px/Plus">
```

Comment:
I'm pretty sure the groups can be removed. Removing Id's is really important, we might end up with repeated ID's in HTML :)

Reply:
Yup, you're right. Changed.